app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,14 @@
+<footer class="footer">
+  <small>
+    The <%= link_to "Ruby on Rails Tutorial", href="https://www.railstutorial.org/" %>
```

Comment:
You shoud install gem config then move "https://www.railstutorial.org/" in setting.yml

Reply:
May I know how to do it in details?

Reply:
Because I follow the instructions from book, so I am not sure how to deal with it.

Reply:
You can follow this instruction 
https://github.com/rubyconfig/config#embedded-ruby-erb
when you installed config gem you have a file name setting.yml, then you can do this 
some_var: "https://www.railstutorial.org/"

<%= link_to "Ruby on Rails Tutorial", Settings.some_var %>

Reply:
Done

Reply:
You have not yet fixed them.
You can follow this link: [Gem config](https://github.com/rubyconfig/config)